### ISSUE TITLE
Voluntary Product Accessibility Template

### ISSUE THREAD

**Issue by LevBHC** [Stellarium/stellarium#1412]
Greetings,

I teach physics and astronomy at Dallas College (Brookhaven Campus).  We use Stellarium as part of our courses.  It has been wonderfully useful and our students have been enjoying working with it.  Recently, due to changes in regulations, the college is requiring that all materials we use are "accessible." As a result of this new regulation, we would require a Voluntary Product Accessibility Template (VPAT) from Stellarium in order that we may continue to use it in our classes.

If you have a Voluntary Product Accessibility Template (VPAT) and Web Content Accessibility Guidelines 2.0 AA Template (WCAG) for your product, please send it my way so that I can submit it for Dallas College accessibility compliance review. Informational links regarding VPAT and WCAG are provided below.

Voluntary Product Accessibility Template (VPAT) - https://www.section508.gov/sell/vpat
Web Content Accessibility Guidelines (WCAG) - https://www.w3.org/WAI/standards-guidelines/wcag/

I look forward to hearing from you!
~Lev

**Comment by gzotti** [comment]
I suggest any regulations intended to classify commercial products offered for sale to US authorities automatically don't apply to non-commercial free open-source products. We don't want to sell anybody anything and don't have to comply to such things even if we may do. And the web thingy by default does IMO not apply to non-web software. 

What are the implications if we declare anything about our free program in terms of the US American "legal system" which most people outside USA entirely fail to understand?  

> This program is distributed in the hope that it will be useful,  but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
